xbmc/interfaces/Builtins.cpp

```diff
@@ -1226,6 +1228,15 @@ int CBuiltins::Execute(const std::string& execString)
     else
       CGUIDialogKaiToast::QueueNotification(params[0],params[1]);
   }
+  else if (execute == "textviewer")
+  {
+    if (params.size() < 2)
+      return -1;
+    CGUIDialogTextViewer* pDlgInfo = (CGUIDialogTextViewer*)g_windowManager.GetWindow(WINDOW_DIALOG_TEXT_VIEWER);
+    pDlgInfo->SetHeading(params[0]);
```

Comment:
you mean something like 

if (pDlgInfo == NULL)
     return;

? (I´m no C programmer).
There is no null check for other similar calls too as far as I can tell.